src/frontend/src/components/ChangeRequestDropdown.tsx

```diff
@@ -0,0 +1,74 @@
+import { MenuItem, Select, SelectChangeEvent, dividerClasses } from '@mui/material';
+import { isWithinInterval, subDays } from 'date-fns';
+import { Control, Controller } from 'react-hook-form';
+import { wbsPipe } from 'shared';
+import { useAllChangeRequests } from '../hooks/change-requests.hooks';
+import LoadingIndicator from './LoadingIndicator';
+import { useCurrentUser } from '../hooks/users.hooks';
+
+interface ChangeRequestDropdownProps {
+  control: Control<any, any>;
+  name: string;
+}
+
+const ChangeRequestDropdown = ({ control, name }: ChangeRequestDropdownProps) => {
+  const { isLoading, data: changeRequests } = useAllChangeRequests();
+  if (isLoading || !changeRequests) return <LoadingIndicator />;
+
+  const today = new Date();
+  const fiveDaysAgo = subDays(today, 5);
+
+  const filteredRequests = changeRequests?.filter(
+    (cr) => !!cr.dateReviewed && isWithinInterval(cr.dateReviewed, { start: fiveDaysAgo, end: today })
+  );
+  // The current user's CRs should be at the top
+  const user = useCurrentUser();
+  filteredRequests.sort((a, b) => {
+    const isSubmitterAUser = a.submitter.firstName === user.firstName && a.submitter.lastName === user.lastName;
+    const isSubmitterBUser = b.submitter.firstName === user.firstName && b.submitter.lastName === user.lastName;
+
+    if (isSubmitterAUser && isSubmitterBUser) return 0;
+    if (isSubmitterAUser) return -1;
+    if (isSubmitterBUser) return 1;
+
+    return a.crId - b.crId;
+  });
```

Comment:
Is it worth making a helper function for this?

Reply:
I would say yes. We can create a helper function that accepts whole change requests from line 16 and returns the filtered and sorted crs.

Reply:
I'll make a function in the same file

Reply:
As long the name of a function specifically explains what it does it will be fine. 

For this case, we can name it as approvedChangeRequests for example.